runtime/lua/vim/lsp/completion.lua

```diff
@@ -507,7 +507,13 @@ local function trigger(bufnr, clients, ctx)
     local server_start_boundary --- @type integer?
     for client_id, response in pairs(responses) do
       if response.err then
-        vim.notify_once(response.err.message, vim.log.levels.WARN)
+        local client = lsp.get_client_by_id(client_id)
```

Comment:
We also get the client [here](https://github.com/neovim/neovim/blob/6f73c62635aa43e4ee0a8a5ec97bd3ac6b7cdc17/runtime/lua/vim/lsp/completion.lua#L522). Could you move this line outside the `response.err` if-block so that we only get the client once?